src/components/AddedFeature.js

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
-
+import {removeFeature} from '../actions/Features'
+import {connect} from 'react-redux';
 const AddedFeature = props => {
+  const removeItem = e =>{
+    e.preventDefault();
```

Comment:
`.preventDefault()` probably isn't needed for an `onClick` since `onClick` doesn't really have a default we would want to prevent, it is more of the `onSubmit` or buttons in side of `<form>` elements that default to `type="submit"` that we would want to `.preventDefault()` for